configs/sites/aws-pcluster/README.md

```diff
@@ -338,6 +347,6 @@ spack install --verbose --source 2>&1 | tee log.install
 spack module lmod refresh
 spack stack setup-meta-modules
 ```
-9. Test spack-stack installation using your favorite application.
+9. Test spack-stack installation using your favorite application. Note that the jedi-bundle ctests requiring MPI don't work correctly, because the modules are configured for SLURM integration but the node in its current state doesn't have SLURM available.
```

Comment:
Note to self. Add note that these tests must be on the provisioned parallel cluster